File: backend/ZJAdmin.Api/Models/ExceptionLog.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ZJAdmin.Api.Models;

[Table("sys_exception_log")]
public class ExceptionLog
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public long Id { get; set; }

    public string? Message { get; set; }

    [MaxLength(200)]
    public string? ExceptionType { get; set; }

    public string? StackTrace { get; set; }

    [MaxLength(500)]
    public string? RequestUrl { get; set; }

    [MaxLength(10)]
    public string? RequestMethod { get; set; }

    public string? RequestParams { get; set; }

    [MaxLength(50)]
    public string? IpAddress { get; set; }

    [MaxLength(50)]
    public string? Operator { get; set; }

    public DateTime OccurTime { get; set; } = DateTime.UtcNow;
}

File: backend-java/src/main/java/com/zjadmin/entity/ExceptionLog.java
package com.zjadmin.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("sys_exception_log")
public class ExceptionLog {

    @TableId(type = IdType.AUTO)
    private Long id;

    private String message;

    private String exceptionType;

    private String stackTrace;

    private String requestUrl;

    private String requestMethod;

    private String requestParams;

    private String ipAddress;

    private String operator;

    private LocalDateTime occurTime;
}
